dev/tasks/conda-recipes/arrow-cpp/meta.yaml

```diff
@@ -61,7 +61,7 @@ outputs:
     build:
       string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
       run_exports:
-        - {{ pin_subpackage("libarrow", max_pin="x.x.x") }}
+        - {{ pin_subpackage("libarrow", max_pin="x") }}
```

Comment:
What's the reason this pin is made smaller? (I am not fully sure if a potential pyarrow 13.0.1 is guaranteed to work with libarrow 13.0.0)

Reply:
See https://github.com/conda-forge/arrow-cpp-feedstock/issues/1096

Reply:
Ah yes, I remember, I actually asked on that issue if we document such guarantee. Personally I am not sure we are properly guaranteeing ABI stability between minor releases